services/payments/datastore.go

```diff
@@ -0,0 +1,386 @@
+package payments
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	errorutils "github.com/brave-intl/bat-go/libs/errors"
+	"github.com/brave-intl/bat-go/libs/logging"
+	appaws "github.com/brave-intl/bat-go/libs/nitro/aws"
+	"github.com/shopspring/decimal"
+
+	"github.com/amazon-ion/ion-go/ion"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/qldbsession"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/aws/smithy-go"
+	"github.com/awslabs/amazon-qldb-driver-go/v3/qldbdriver"
+	appctx "github.com/brave-intl/bat-go/libs/context"
+	"github.com/google/uuid"
+)
+
+// Transaction - the main type explaining a transaction, type used for qldb via ion
+type Transaction struct {
```

Comment:
In the tooling there is a [version](https://github.com/brave-intl/bat-go/blob/4a3b5b79971c7d2e6aa233e8b1c52853ddc0c77d/tools/payments/transaction.go#L85) but it is not defined here. Do we need to add or remove from tooling?